### ISSUE TITLE
rabbitmqctl doesn't support arguments with a ` in them

### ISSUE THREAD

**Issue by ben-page** [rabbitmq/rabbitmq-server#313]
When rabbitmqctl extracts it's args it doesn't properly quote them before passing them along to rabbitmq. Therefor you can not create a password with a ` in it.

```shell
sudo rabbitmqctl add_user test '`pass'
sh: 1: Syntax error: EOF in backquote substitution
```

I fixed the script and created a pull request.

**Comment by ben-page** [comment]
https://github.com/rabbitmq/rabbitmq-server/pull/314

**Comment by michaelklishin** [comment]
Closed in #318, will be in `3.5.5`. Thank you, @ben-page!
